.pre-commit-config.yaml

```diff
@@ -12,31 +12,6 @@ repos:
       - id: detect-private-key
       - id: check-shebang-scripts-are-executable
 
-  - repo: https://github.com/adrienverge/yamllint.git
-    rev: v1.32.0
-    hooks:
-      - id: yamllint
-        entry: yamllint --strict -c .hooks/linters/yamllint.yaml
-
-  - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.0.3
-    hooks:
```

Comment:
no more .json files in this repo now that renovate is gone, just some vscode configs that will probably get removed later anyway